src/ert/dark_storage/endpoints/records.py

```diff
@@ -27,117 +26,11 @@
 DEFAULT_HEADER = Header("application/json")
 
```

Comment:
So all these end points were not in use?

Reply:
They were not in use